docs/installation/installation.md

```diff
@@ -229,3 +229,29 @@ member3   v1.23.4   Pull   True    7m27s
 ```
 
 There are 3 clusters named `member1`, `member2` and `member3` have registered with `Push` or `Pull` mode.
+
+## Required Images for Karmada Deployment
+
+The following table lists the images required for deploying Karmada:
+
+| Component                         | Default Image Registry + Name                    | Default Tag                 |
+| --------------------------------- | ------------------------------------------------ |-----------------------------|
+| **Karmada API Server**            | `registry.k8s.io/kube-apiserver`                 | `v1.29.6` (current release) |
+| **Karmada Aggregated API Server** | `docker.io/karmada/karmada-aggregated-apiserver` | `v${karmadaGitVersion}`     |
+| **Kube Controller Manager**       | `registry.k8s.io/kube-controller-manager`        | `v1.30.4` (current release) |
+| **Karmada Controller Manager**    | `docker.io/karmada/karmada-controller-manager`   | `v${karmadaGitVersion}`     |
+| **Karmada Scheduler**             | `docker.io/karmada/karmada-scheduler`            | `v${karmadaGitVersion}`     |
+| **Karmada Webhook**               | `docker.io/karmada/karmada-webhook`              | `v${karmadaGitVersion}`     |
+| **Etcd (Local)**                  | `registry.k8s.io/etcd`                           | `latest`  (current release) |
+| **Etcd Init Image (Local)**       | `docker.io/alpine`                               | `3.19.1`  (current release) |
+| **Karmada Agent**                 | `docker.io/karmada/karmada-agent`                | `v${karmadaGitVersion}`     |
```

Comment:
See the comments https://github.com/karmada-io/website/pull/690#discussion_r1818434676.

Reply:
Yes, I accidentally missed that change 😭. Now, that part has been updated in the documentation, and I look forward to further progress.

Reply:
> See the comments [#690 (comment)](https://github.com/karmada-io/website/pull/690#discussion_r1818434676).

Yes, I accidentally missed that change 😭. Now, that part has been updated in the documentation, and I look forward to further progress.